metricbeat/module/vsphere/resourcepool/_meta/fields.yml

```diff
@@ -0,0 +1,85 @@
+- name: resourcepool
+  type: group
+  release: beta
+  description: >
+    resourcepool
+  fields:
+    - name: cpu
+      type: group
+      fields:
+        - name: entitlement
+          type: group
+          fields:
+            - name: mhz
+              type: long
+              description: >
+                Amount of CPU resources allocated to the resource pool.
+        - name: usage
+          type: group
+          fields:
+            - name: pct
+              type: long
+              description: >
+                CPU usage average in percentage.
+            - name: mhz
+              type: long
+              description: >
+                CPU usage average in MHz.
+        - name: active
+          type: group
+          fields:
+            - name: average
+              type: group
+              fields:
+                - name: pct
```

Comment:
Do you think we can use `format: percent` for percentage fields? [Refer](https://github1s.com/elastic/beats/blob/main/x-pack/metricbeat/module/awsfargate/task_stats/_meta/fields.yml#L35-L36)